source/nodejs/adaptivecards-react/package.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "adaptivecards-react",
+	"version": "2.10.0-beta.1",
```

Comment:
Our current practice is to version each package independently, following semver. So, this should probably be something like `0.1.0`, `1.0.0-beta.1`, or `1.0.0`. Thoughts? @jonmill @dclaux @paulcam206

Reply:
Agreed. 1.0.0-beta.1 sounds good.

Reply:
I'd like our semver to actually have meaning & follow the intent of semver...meaning that the major and minor versions should indicate functionality (packages with major v6 should have similar functionality) and the patch version differs based on bug fixes. I'm okay with this package playing catch up since it uses the underlying JS package and provides a bridge between vanilla JS and React

Reply:
@jonmill are you saying go with `2.10.0-beta.1` or something along the lines of what Risheek is proposing?

Reply:
My recommendation is to follow the JS semver to make it follow the intent of semver (where major + minor describes what functionality is available).

That being said, this isn't something I feel strongly about and can go either direction if folks feel otherwise🙂

Reply:
Oh you mean follow the JS SDK's version?

If so I'm not sure I agree. The React component package's dependency on the adaptivecards package is what should establish the relationship IMO. I foresee that we will ship multiple new versions of the adaptivecards SDK itself while the ReactJS component package will remain at its current version because we made no change to it.

Reply:
Sounds good to me 🙂 feel free to follow David and Risheek's suggestion for `1.0.0-beta1`

Reply:
@marakim - feel free to update to as described here :)

Reply:
Updated to change version to `1.0.0-beta1`